### DISCUSSION TITLE
encounter this error message when try using azure auth

### DISCUSSION THREAD

**Discussion by wenqvip** [outline/outline#2463] [Self Hosting] [answered]
> AADSTS500201: We are unable to issue tokens from this API version for a Microsoft account. Please contact the application vendor as they need to use version 2.0 of the protocol to support this.

I followed the direction [Microsoft Setup](https://wiki.generaloutline.com/share/dfa77e56-d4d2-4b51-8ff8-84ea6608faa4) to do every step on it.

I am using docker to launch the outline and set .env file. 
Azure information is filled
```
AZURE_CLIENT_ID=b4ae551b-a63e-41e2-b12f-xxxxxxxxxxx
AZURE_CLIENT_SECRET=88a55077-e3ea-428a-8a9b-xxxxxxxxxxxxx
AZURE_RESOURCE_APP_ID=00000003-0000-0000-c000-000000000000
```
I have got frustrated about this, need help

**Comment by tommoor** [comment, marked as answer]
I've never seen this error before, what type of Microsoft account are you using?

**Reply by wenqvip** [reply]
I select "Accounts in this organizational directory only" as the guide document say.

**Reply by tommoor** [reply]
What type of Microsoft organization is it though?

**Reply by wenqvip** [reply]
I use my private account so I guess the organization is private too ?

**Reply by tommoor** [reply]
I suspect that's the issue, the account must be one of:

- Office365
- Azure AD
- Exchange

**Reply by wenqvip** [reply]
thanks! but I tried auth with slack, it works for now. Maybe I try azure again some day
